### ISSUE TITLE
Crashes when rendering ImageComposeScene

### ISSUE THREAD

**Issue by jimgoog** [JetBrains/compose-jb#1392]
Code in question crashes in a variety of inconsistent ways (core dump, stack traces, native code errors, etc):
```
fun main() {
    val scene = ImageComposeScene(100, 100) {
        CircularProgressIndicator()
    }
    for (i in 1..10000) {
        scene.render(Duration.milliseconds(i)).close()
    }
}
```

cc @igordmn

**Comment by igordmn** [comment]
Should be fixed in 1.1.0

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6118">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
